docker/cygnus-ngsi/Dockerfile

```diff
@@ -65,9 +67,9 @@ RUN yum -y install tar nc which git java-1.6.0-openjdk-devel && \
 # Create Cygnus log folder
 RUN mkdir ${FLUME_HOME}/log
 
-# Clone fiware-cygnus
-RUN git clone ${GIT_URL_CYGNUS} ${CYGNUS_HOME} && \
-    cd ${CYGNUS_HOME} && \
+# Instead of re-cloning, copy the fiware-cygnus repo, move there and checkout the propper branch
+COPY . ${CYGNUS_HOME}
```

Comment:
I guess @dmoranj suggested to avoid the clone because it may take a while to clone again, specially after having cloned for getting access to this `Dockerfile` file.

Regarding @albertinisg comment, it seems dockerhub performs a clone by itself... the problem is it executes `docker build .` from the folder containing `Dockerfile`, so it is possible it does not work, yes.